saleor/discount/utils.py

```diff
@@ -186,6 +186,21 @@ def get_voucher_code_instance(
     return code_instance
 
 
+def get_active_voucher_code(voucher, channel_slug):
```

Comment:
I added utils to get `VoucherCode`, but I don't know if this one is still required as a separate method.